frontend/app/work_packages/directives/work-package-comment-directive.js

```diff
@@ -114,7 +114,7 @@ module.exports = function(
         ctrl.writeValue
       ).then(function() {
         ctrl.discardEditing();
-        NotificationsService.addSuccess(I18n.t('js.work_packages.comment_added'));
+        NotificationsService.addSuccess(I18n.t('js.notice_successful_update'));
```

Comment:
Why did you change this line? If I understand the specification correctly, the comment success notification should be removed